examples/argocd/README.md

```diff
@@ -45,73 +45,77 @@ The following command will update the `kubeconfig` on your local machine and all
 1. Run `update-kubeconfig` command:
 
     ```sh
-    aws eks --region <REGION> update-kubeconfig --name <CLUSTER_NAME>
+    aws eks --region <REGION> update-kubeconfig --name <CLUSTER_NAME> --alias <CLUSTER_NAME>
```

Comment:
nit: What is the motivation for the `--alias` flag? Only reason I ask is for consistency - if we are going to use it in the examples, then we should use it across all of the examples as well as the generated output

Reply:
If you don't have the `--alias` it add the cluster to your context with the full ARN and when you are switching contexts it is unnecessary convoluted. `--alias` is not a necessity but definitely recommend to adopt it as a pattern to the rest of the examples.